app/src/main/java/com/practicum/playlistmarket/player/data/repository/PlayerRepositoryImpl.kt

```diff
@@ -107,17 +100,16 @@ class PlayerRepositoryImpl(
         )
     }
 
-    companion object {
-        private const val REFRESH_LIST_DELAY_MILLIS = 200L
-        private const val DEFAULT_TIME_TRACK = "00:00"
+
+   override fun setupListener(listener: PlayerListener){
```

Comment:
```suggestion
   override fun setupListener(listener: PlayerListener?) {
```
Лучше разрешить передавать null в случае, если вызывающая сторона захочет отписаться от событий